boards/cc1312-launchpad/include/periph_conf.h

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (C)    2016 Nicholas Jackson
+ *                  2017 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup         boards_cc1312_launchpad
+ * @{
+ *
+ * @file
+ * @brief           Peripheral MCU configuration for TI CC1312 LaunchPad
+ *
+ * @author          Nicholas Jackson <nicholas.jackson@griffithuni.edu.au>
+ * @author          Sebastian Meiling <s@mlng.net>
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include "periph_cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+* @name    Clock configuration
+* @{
+*/
+/* the main clock is fixed to 48MHZ */
+#define CLOCK_CORECLOCK     (48000000U)
+/** @} */
+
+/**
+* @name    Timer configuration
+*
+* General purpose timers (GPT[0-3]) are configured consecutively and in order
+* (without gaps) starting from GPT0, i.e. if multiple timers are enabled.
+*
+* @{
+*/
+static const timer_conf_t timer_config[] = {
+ {
+     .cfg = GPT_CFG_16T,
+     .chn = 2,
+ },
+ {
+     .cfg = GPT_CFG_32T,
+     .chn = 1,
+ },
+ {
+     .cfg = GPT_CFG_16T,
+     .chn = 2,
+ },
+ {
+     .cfg = GPT_CFG_32T,
+     .chn = 1,
+ }
+};
+
+#define TIMER_NUMOF         ARRAY_SIZE(timer_config)
+/** @} */
+
+/**
+* @name    UART configuration
+*
+* The used CC1312 CPU only supports a single UART device, so all we need to
+* configure are the RX and TX pins.
+*
+* Optionally we can enable hardware flow control, by setting UART_HW_FLOW_CTRL
+* to 1 and defining pins for UART_CTS_PIN and UART_RTS_PIN.
+* @{
+*/
+/**
+ * @name    UART configuration
```

Comment:
I wondering how this renders in the documentation. I think the 2 `@name  UART configuration` blocks should be merged.

Reply:
You're right, they should be merged. [See this](https://doc.riot-os.org/boards_2cc1352-launchpad_2include_2periph__conf_8h.html), it's how that block is rendered for the `cc1352-launchpad` board, it needs to be fixed too. Maybe for another PR, alongside the `export` that is too on that board.